keras/applications/applications_test.py

```diff
@@ -76,7 +77,12 @@
                         (regnet.RegNetY016, 888), (regnet.RegNetY032, 1512),
                         (regnet.RegNetY040, 1088), (regnet.RegNetY064, 1296),
                         (regnet.RegNetY080, 2016), (regnet.RegNetY120, 2240),
-                        (regnet.RegNetY160, 3024), (regnet.RegNetY320, 3712)]
+                        (regnet.RegNetY160, 3024), (regnet.RegNetY320, 3712),
+                        (resnet_rs.ResNetRS50,2048),(resnet_rs.ResNetRS101,2048),
```

Comment:
Thanks for the PR and your patience.  Sorry, this fell through the cracks on my end for a bit.

add spaces after commas in line 82-84

Reply:
Done